oneanddone/tasks/filters.py

```diff
@@ -43,6 +43,11 @@ class TasksFilterSet(django_filters.FilterSet):
         label=_lazy(u'Search for tasks')
     )
 
+    difficulty = django_filters.MultipleChoiceFilter(
+        choices=((Task.BEGINNER, 'Beginner'), (Task.INTERMEDIATE, 'Intermediate'), (Task.ADVANCED, 'Advanced')),
```

Comment:
This line is over 100 characters long, which is far too long. I suggest breaking it up like so:

```
        choices=(
            (Task.BEGINNER, 'Beginner'),
            (Task.INTERMEDIATE, 'Intermediate'),
            (Task.ADVANCED, 'Advanced')),
```